### PR TITLE
Create Blog “2019-03-28-how-to-avoid-dirty-reads-with-event-store”

### PR DESCRIPTION
**Pull request by riccardone**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
Started an edit here @riccardone I have one question…

File: _posts/2019-03-28-how-to-avoid-dirty-reads-with-event-store.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+In other words in a cluster of 3 nodes the quorum is 2 nodes. Before the Master can send back the acknowledge to the client it needs to wait the confirmation of write from at least 1 Slave node. For that reason **it is possible that a client can read from the other node that is not updated yet with the latest write**. This situation is defined as “Dirty Read”.
+
+If you must ensure that a client will always read the latest version of your data then you can...
+
+If you use the .net client api: use the connection setting\
+`PerformOnMasterOnly`
+
+If your client doesn’t provides this method use the connection string setting \
+`require-master=true`
```
Comment (ChrisChinchilla) [inline _posts/2019-03-28-how-to-avoid-dirty-reads-with-event-store.md]: @riccardone I can't find this in the codebase or the docs, any more details?

Comment (riccardone) [reply _posts/2019-03-28-how-to-avoid-dirty-reads-with-event-store.md]: Most of the conn settings can be specified customising the connection string like:
"ConnectTo=tcp://admin:changeit@localhost:1113; HeartBeatTimeout=500;require-master=true"

https://eventstore.org/docs/dotnet-api/connecting-to-a-server/index.html#customising-connection-settings

Comment (ChrisChinchilla) [reply _posts/2019-03-28-how-to-avoid-dirty-reads-with-event-store.md]: Ahh, the hyphen threw me, OK, cool 👍

**Pull request merged**